Modules/socketmodule.c

```diff
@@ -7799,6 +7811,11 @@ socket_exec(PyObject *m)
 #endif /* BTPROTO_RFCOMM */
     ADD_STR_CONST(m, "BDADDR_ANY", "00:00:00:00:00:00");
     ADD_STR_CONST(m, "BDADDR_LOCAL", "00:00:00:FF:FF:FF");
+#ifdef BDADDR_BREDR
+    ADD_INT_MACRO(m, BDADDR_BREDR);
+    ADD_INT_MACRO(m, BDADDR_LE_PUBLIC);
+    ADD_INT_MACRO(m, BDADDR_LE_RANDOM);
```

Comment:
Please add these to `Doc/library/socket.rst`, see existing `BDADDR_ANY` docs for an example.